pkgs/development/libraries/libgit2/default.nix

```
@@ -11,6 +11,7 @@
 , http-parser
 , libiconv
 , Security
+, static ? stdenv.hostPlatform.isStatic
```

```suggestion
, staticBuild ? stdenv.hostPlatform.isStatic
```

A better name for this variable.